beets/plugins.py

```diff
@@ -488,3 +488,13 @@ def sanitize_choices(choices, choices_all):
             if not (s in seen or seen.add(s)):
                 res.extend(list(others) if s == '*' else [s])
     return res
+
+
+def notify_info_received(msg):
```

Comment:
Well I totally agree that it is not ideal as is though. Since this is a WIP I'll keep that in the back of my head as a continue examining other things.